ECMA2Yaml/IntellisenseFileGen/Program.cs

```diff
@@ -202,7 +203,10 @@ public static List<Models.Type> LoadTypes(Dictionary<string, ECMA2Yaml.Models.Re
                     Models.Type t = ConvertToType(xmlDoc, ItemsByDocId);
```

Comment:
use `ConcurrentBag<Models.Type> typeList = new ConcurrentBag<Models.Type>();` then you don't have to use lock